portal/config/gil/site_persistence_file.json

```diff
@@ -20,7 +20,6 @@
         "TRUENTH_LINK_URL='https://us.movember.com/'\n",
         "MOVEMER_LINK_URL='https://us.movember.com/'\n",
         "STAFF_BULK_DATA_ACCESS=False\n",
-        "ALLOW_STAFFADMIN_DELEGATION = True\n",
```

Comment:
Does this config also need to be deleted from this template?

https://github.com/uwcirg/true_nth_usa_portal/blob/develop/portal/templates/profile/staff_profile_create.html#L19

Reply:
@ivan-c ivan, thank u very much for catching that.